app/Http/Kernel.php

```diff
@@ -49,53 +52,43 @@ class Kernel extends HttpKernel
         ConvertEmptyStringsToNull::class,
         Middleware\TrustProxies::class,
     ];
+
     /**
      * The application's route middleware groups.
      */
     protected $middlewareGroups = [
-        'web' => [
-            Middleware\EncryptCookies::class,
-            AddQueuedCookiesToResponse::class,
-            StartSession::class,
-            // \Illuminate\Session\Middleware\AuthenticateSession::class,
-            ShareErrorsFromSession::class,
-            Middleware\VerifyCsrfToken::class,
-            SubstituteBindings::class,
+        self::WEB => [
+            'cors',
         ],
 
-        'api' => [
-            // 'throttle:60,1',
-            HandleCors::class,
-            'auth:api',
+        self::APP => [
+            'cors',
+            'session',
             'bindings',
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "bindings" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)